biz_portal/apps/portal/tests/test_admin.py

```diff
@@ -268,3 +293,45 @@ def test_superuser_can_export_businesses(self):
             HTTP_HOST="biz-portal.openup.org.za",
         )
         self.assertEqual(response.status_code, 200)
+
+
+class AdminBulkLoadDirectorsTestCase(TestCase):
+    """Test for Bulk loads"""
+
+    fixtures = [
+        "sectors",
+        "business_types",
+        "business_statuses",
+        "regions",
+        "groups",
+        "test_api_businesses",
+        "test_bulk_load",
+    ]
+
+    def test_bulk_load_directors_correctly_match(self):
+        """It verifies bulk upload functionality including correct matching of business by its registration number"""
+        self.assertTrue(self.client.login(username="admin", password="password"))
+        self.assertEqual(models.BusinessMembership.objects.count(), 0)
+
+        data = tablib.Dataset()
+        data.headers = [
+            "id",
```

Comment:
what is this id field? it shoudn't be here?

Reply:
I removed it. It's a mistake